KubeArmor/monitor/syscallParser.go

```diff
@@ -1108,3 +1108,14 @@ func GetArgs(dataBuff *bytes.Buffer, Argnum int32) ([]interface{}, error) {
 
 	return args, nil
 }
+
+var auditedSyscalls = map[int]string{
+	87: "unlink",
+}
```

Comment:
We extend to add further syscalls here in a subsequent PR. But do we plan to Audit existing syscalls?

Like we classify sockert/connect/accept/bind as Network, but what if I want to audit a a particular syscall here. This is out of scope of the PR but wanted to understand future plans.

Reply:
Yes we are planing to, but we will not audit all system calls. We will audit only for a subset of them.